src/api.ts

```diff
@@ -21,4 +21,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId: number | undefined) => get<User>(`/users/${userId}`);
```

Comment:
```suggestion
export const getUser = (userId?: number) => get<User>(`/users/${userId}`);
```

Reply:
Btw, should we send request if id is undefined? Now it will send request to endpoint '/users/undefined' and it looks strange